package.json

```diff
@@ -4,22 +4,25 @@
   "description": "Opinionated logging framework based on @dbuenzli/logs",
   "license": "MIT",
   "esy": {
-    "build": "refmterr dune build -p timber -j4"
+    "build": "dune build -p timber -j4",
+    "buildDev": "refmterr dune build -p timber -j4"
   },
   "scripts": {
     "format": "esy dune build @fmt --auto-promote"
   },
   "dependencies": {
-    "@esy-ocaml/reason": "3.5.2",
-    "refmterr": "^3.3.0",
     "@opam/dune": "*",
-    "ocaml": "^4.6.0",
     "@opam/fmt": "^0.8.8",
     "@opam/logs": "^0.7.0",
-    "@opam/re": "*"
+    "@opam/re": "*",
+    "@reason-native/console": ">0.0.0",
```

Comment:
This isn't actually needed anymore. The reference in the dune file should be removed as well.